entity/src/main/java/com/wilma/entity/users/Resume.java

```diff
@@ -0,0 +1,53 @@
+package com.wilma.entity.users;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.Date;
+
+
+@Getter
+@Setter
+@AllArgsConstructor
+@Entity
+@Table(name = "files")
+public class Resume {
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "uuid2")
+    private String id;
```

Comment:
This can't work
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id", nullable = false)
    private Integer id;
```